SEP-0000.md

```diff
@@ -0,0 +1,29 @@
+# SEP-0000 -- Index of SunPy Enhancement Proposals (SEPs)
+
+| SEP           | 0 |
```

Comment:
I'm not a fan of using the formatting this way because "SEP" and "0" are rendered as column headers, which they aren't.  Perhaps just remove the row entirely since the SEP number is already visible in the title?